test/src/send.test.js

```diff
@@ -62,6 +62,13 @@ contract('send', function ([sender, receiver]) {
         );
       });
 
+      it('calls fallback function', async function () {
+        const receipt = await send.transaction(this.acknowledger, '', '', [], opts);
```

Comment:
I'd rather we were able to avoid having to pass an empty string. If only JavaScript supported named arguments...
Perhaps we could have `send.transaction` read `opts` from the last value in `arguments`? @frangio

Reply:
I don't understand, we already have `send.ether` for this.

Reply:
Right. So we should still support the contract having a fallback function, but calling it should be done via `send.ether`.